apache-maven/src/assembly/maven/conf/maven.properties

```diff
@@ -0,0 +1,40 @@
+#
+# Maven user properties
+#
+
+maven.installation.conf = ${maven.home}/conf
+maven.user.conf         = ${user.home}/.m2
+maven.project.conf      = ${session.rootDirectory}/.mvn
+
+# Comma-separated list of files to include.
+# Each item may be enclosed in quotes to gracefully include spaces. Items are trimmed before being loaded.
+# If the first character of an item is a question mark, the load will silently fail if the file does not exist.
+${includes} = "?${maven.user.conf}/maven.properties", \
```

Comment:
Since you are quoting now the value, wouldn't it be more natural to put the question mark at the end? Unless you want to use "¿" as well. Maybe qoutes should be mandatory. Worth looking into Tomcat code for ths. I do not remember the motiviation.

Reply:
I'm not sure quotes are mandatory, but they certainly avoid any problem with spaces inside path.  
About the location of the '?', I fear that putting it at end will be less prominent.
It's also the same syntax than for optional profiles, see https://maven.apache.org/guides/introduction/introduction-to-profiles.html#explicit-profile-activation

Reply:
I see your point. Shouldn't then the question mark be outside of the quotation marks to make the entire space-containing value optional?

Reply:
Yes, makes sense.

Reply:
I've added support for moving the '?' before the quotes (both before and after are supported), the default config file moves them before.